todo/Controllers/AccountsController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Todo.DTOs;
+using Todo.Models;
+using Todo.Services;
+
+namespace Todo.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AccountsController : ControllerBase
+    {
+        private readonly IAccountService accountRepo;
```

Comment:
```suggestion
        private readonly IAccountService accountService;
```
sửa thì nhớ sửa hết nhé